apps/omg_db/lib/omg_db/rocksdb/core.ex

```diff
@@ -0,0 +1,132 @@
+# Copyright 2018 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.DB.RocksDB.Core do
+  @moduledoc """
+  Responsible for converting type-aware, logic-specific queries and updates into leveldb specific queries and updates
+  """
+
+  # adapter - testable, if we really really want to
+
+  @single_value_parameter_names OMG.DB.single_value_parameter_names()
+
+  @keys_prefixes %{
```

Comment:
I'm a bit nervous about these prefixes. Let's sanitize them a bit, when we have the clean chance. I was thinking about this, but never wanted to plunge into. I suggest the following:

```
# two-letter prefixes always for various buckets of data
@keys_prefixes %{
    block: "bl",
    block_hash: "bh",
    utxo: "ut",
    exit_info: "ei",
    in_flight_exit_info: "if",
    competitor_info: "ci",
    spend: "sp"
  }
```

what do you think?